content/en/docs/tasks/administer-cluster/migrating-from-dockershim/_index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Migrating from dockershim to other container runtimes"
+weight: 10
+---
+
+<!-- overview -->
+
+This section presents information you need to know when migrating from
+dockershim to other container runtimes.
+
+Since the announcement of [dockershim deprecation] in Kubernetes 1.20, there
+were questions on how this will affect various workloads and Kubernetes
+installations. You can find these links useful to understand the problem better:
+
+- [Don't Panic: Kubernetes and Docker](/blog/2020/12/02/dont-panic-kubernetes-and-docker/)
+- [Dockershim Deprecation FAQ](https://kubernetes.io/blog/2020/12/02/dockershim-faq/)
+
+We highly encourage you to consider various migration options. Check out
```

Comment:
If you can, [avoid using “we”](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-using-we).

Reply:
fixed. thx